### PR TITLE
Remove FIG format, add DITAA

### PR DESCRIPTION
**Pull request by sknorr**
I am sure these make sense:
* FIG is no longer supported with DAPS 3.1 and above
* likewise EPS and PDF are no longer supported with DAPS 3.1 and above

I am not sure about the second commit:
* DITAA is newly supported in DAPS 3.1 -- however, it is _not_ an output format, it is an input format only. I wonder if it makes any sense to list all these input-only formats.

As an addendum, I am wondering if it makes sense to list both "jpg" and "jpeg". As far as I can see, DAPS only supports "jpg". I could add a third commit to fix that too.

### CODE REVIEW

File: geekodoc/rng/2_5.2/geekodoc-v2.rnc [hunk 1]
```diff
@@ -135,22 +135,20 @@ div {
...
      "EPS" |
      ## Encapsulated PostScript
      "eps" |
-     ## Format of the xfig tool
-     "FIG" |
-     ## Format of the xfig tool
-     "fig" |
+     ## Format of the DITAA tool
+     "DITAA" |
+     "ditaa" |
```
Comment (tomschr) [inline geekodoc/rng/2_5.2/geekodoc-v2.rnc:147]: Why does this occurs twice? It's already in line 138-140.

**Review by tomschr — CHANGES_REQUESTED** [review]
Thanks @sknorr, very appreciated!

Just a minor comment, otherwise it looks good. Maybe we should add ditaa to the list.

File: geekodoc/rng/2_5.2/geekodoc-v2.rnc (hunk 1 above)
Comment (sknorr) [reply geekodoc/rng/2_5.2/geekodoc-v2.rnc:147]: Indeed that should be fixed before merging. However, on the general point I made in the description regarding listing all of those input formats ... what do you say?

**Comment by tomschr** [comment]
Yes, FIG and PDF make sense to remove them.

Regarding the DITAA format: I'd say, let's add it. It shouldn't hurt.

**Comment by sknorr** [comment]
>Regarding the DITAA format: I'd say, let's add it. It shouldn't hurt.

So the reason I was asking is that from what I understood, this attribute is supposed to capture the _target conversion format_. Is that correct?

**Comment by sknorr** [comment]
We have discussed this and it's probably best to just remove the format attribute entirely.

**Comment by sknorr** [comment]
Follow-up: #93 ->

**Pull request closed without merging**